kevlar/simlike.py

```diff
@@ -252,6 +252,11 @@ def simlike(variants, case, controls, refr, mu=30.0, sigma=8.0, epsilon=0.001,
         abovethresh = [a for a in altabund[0] if a > casemin]
         if len(abovethresh) == 0:
             call.filter(kevlar.vcf.VariantFilter.PassengerVariant)
+        for abundlist in altabund[1:]:
+            toohigh = [a for a in abundlist if a > ctrlmax]
+            if len(toohigh) > 2:
```

Comment:
Probably best to go with the more conservative `if len(toohigh) > 4`. Or make it a tunable parameter.